src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/ShowPopup/readme.md

```diff
@@ -0,0 +1,39 @@
+# Show a popup
+
+Show predefined popups from a web map.
+
+![Show a popup screenshot](ShowPopup.jpg)
+
+## Use case
+
+Many web maps contain predefined popups which are used to display the attributes associated with each feature layer in the map, such as hiking trails, land values, or unemployment rates. You can display text, attachments, images, charts, and web links. Rather than creating new popups to display information, you can easily access and display the predefined popups. 
```

Comment:
Looking at the implementation of the popup viewer, I don't think we support anything other than plain field label+value display, or plaintext html descriptions (https://github.com/Esri/arcgis-toolkit-dotnet/blob/main/src/Toolkit/Toolkit.WPF/UI/Controls/PopupViewer/PopupViewer.Theme.xaml, https://github.com/Esri/arcgis-toolkit-dotnet/blob/98acce57f73197f5abf27df58c7d82838323a27d/src/Toolkit/Toolkit/Internal/StringExtensions.cs) - it would be worth clarifying that as written, the sample & toolkit don't support rich popups.